soc/arm/atmel_sam/same70/Kconfig.defconfig.series

```diff
@@ -84,4 +84,8 @@ config PWM_SAM
 	default y
 	depends on PWM
 
+config CAN_SAM
+	default y
+	depends on CAN
+
```

Comment:
Please, add this to samv71 too.

Reply:
I'll try to use the "new way" to do it. See STM32 implementation